### ISSUE TITLE
Add ``atol`` argument to function ``is_O3``

### ISSUE THREAD

**Issue by nstarman** [astropy/astropy#13694]
https://github.com/astropy/astropy/blob/3912916dad56920514ba648be400a5f82add041a/astropy/coordinates/matrix_utilities.py#L137-L163

**Comment by pllim** [comment]
Sounds like a reasonable request, especially given you were the one who added it in https://github.com/astropy/astropy/pull/11444 . 😸

**Comment by nstarman** [comment]
Yeah 😆 . An effective, if somewhat incompletely-implemented solution.
